worlds/factorio/Options.py

```diff
@@ -146,8 +146,8 @@ class TechTreeLayout(Choice):
 
 class TechTreeInformation(Choice):
     """How much information should be displayed in the tech tree.
-    None: No indication what a research unlocks
-    Advancement: Indicators which researches unlock items that are considered logical advancements
+    None: No indication of what a research unlocks
+    Advancement: Indicates which researches unlock items that are considered logical advancements, but not who they are for
```

Comment:
```suggestion
    None: No indication of what a research unlocks.
    Advancement: Indicates which researches unlock items that are considered logical advancements, but not who they are for.
```

To be more consistent with the "Full" option line.